pkgs/development/python-modules/scikit-image/default.nix

```diff
@@ -43,12 +52,41 @@ buildPythonPackage rec {
 
   checkInputs = [ pytest ];
 
-  # No tests in archive
-  doCheck = false;
+  # See the explanation above
+  checkPhase = ''
+    rm -r skimage
+    export INSTALL_PATH=$out/${python.sitePackages}
+    pytest $INSTALL_PATH \
+      --ignore=$INSTALL_PATH/skimage/filters/rank/tests/test_rank.py \
```

Comment:
It would be good to include reasons for the disabled tests, otherwise it just looks like we're disabling the ones that don't pass, which rather defeats the point ;)

Reply:
Of course, have forgot about it. These tests require network connection, will add a comment.